grafana/overlays/osc/osc-cl1/datasources/grafanadatasource.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: integreatly.org/v1alpha1
+kind: GrafanaDataSource
+metadata:
+  name: datasource
+spec:
+  datasources:
+    - editable: false
+      jsonData:
+        httpHeaderName1: Authorization
+        timeInterval: 5s
+        tlsSkipVerify: true
+      name: emea-morty
+      secureJsonData:
+        httpHeaderValue1: $UWM_BEARER_TOKEN
+      type: prometheus
+      url: https://thanos-querier.openshift-monitoring.svc.cluster.local:9091
```

Comment:
is this the right thanos instance as a datasource? Shouldnt it be the frontend-queirier?

Reply:
[nope](https://console-openshift-console.apps.odh-cl1.apps.os-climate.org/k8s/ns/openshift-monitoring/routes/thanos-querier)